requirements-wheels.txt

```diff
@@ -1,4 +1,5 @@
 click == 8.1.*
+coverage == 7.0.*
```

Comment:
Could this just be a dev dependency?

Reply:
That's the goal, yes. ATM dev dependencies are not used in the python_tests action, so I need to look into modify the dockerfile.
Before that I need to figure out how to get the .coverage file out of the container so that it can be uploaded to codecov...